src/js/clickable-component.js

```diff
@@ -130,6 +130,30 @@ class ClickableComponent extends Component {
   }
 
   /**
+   * Enable the component element
+   *
+   * @return {Component}
+   * @method enable
+   */
+  enable() {
+    this.removeClass('vjs-disabled');
+    this.el_.setAttribute('aria-disabled', 'false');
```

Comment:
:+1: That would be a good place to check that we're consistently passing 'false' or 'true' (strings) to ARIA attributes, rather than booleans, which I *think* causes problems.